src/ResourceManagement/AzureStackStorageAdmin/NuGet.Config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<settings>
+  <repositoryPath>..\..\packages</repositoryPath>
```

Comment:
Align this folder path with the rest of projects in the ResourceManagement folder. I think you are missing one more ..\